### DISCUSSION TITLE
Sequelize.TIME(0) ignored in MSSQL?

### DISCUSSION THREAD

**Discussion by notflip** [sequelize/sequelize#15721] [Help & Questions] [answered]
Does anyone know if this is normal behaviour?
I'm trying to add `time_field: { type: Sequelize.TIME(0) },` but when I look inside the database after the migration the field says `TIME(7)`.

Am I missing something? Thank you!

**Comment by ephys** [comment, marked as answer]
From what I can see, `DataTypes.TIME` does not take any options: https://github.com/sequelize/sequelize/blob/d3f5b5a65e297f4b6861e6a6ce335a9830b28781/src/data-types.js#L416-L420

The precision option is only available since Sequelize 7 https://github.com/sequelize/sequelize/blob/58c333afa9cc2da36e7b9650fab1982c8019a04f/packages/core/src/dialects/abstract/data-types.ts#L1335-L1366

**Reply by notflip** [reply]
Thanks! That makes sense.
